### ISSUE TITLE
Property name 'stormpathMigrationRecoveryAnswer' is not defined in profile

### ISSUE THREAD

**Issue by mraible** [stormpath/stormpath-sdk-java#1335]
When I try to register a new user using 2.0.0-okta-rc1, I get an error.

```
Property name 'stormpathMigrationRecoveryAnswer' is not defined in profile
```

![stormpathmigrationrecoveryanswer](https://cloud.githubusercontent.com/assets/17892/26685478/81033a2e-46a7-11e7-93c5-449fb997bd16.png)

I did not migrate a Stormpath tenant to Okta, I'm just using an Okta account I created myself. Maybe that's the problem?

**Comment by bdemers** [comment]
Yes, the migration tool will create that property, you can create them manually too:
see https://github.com/stormpath/stormpath-sdk-java/blob/okta/OktaGettingStarted.md#required-okta-configuration-change
